Python 3.7 build linter issues

From [linter warnings](https://github.com/Algebra8/LAViewSet/runs/1449804801?check_suite_focus=true) for Python3.7 build:

```
./build/lib/laviewset/routes.py:138:1: W391 blank line at end of file
./build/lib/laviewset/__init__.py:1:1: F401 '.routes.Route' imported but unused
./build/lib/laviewset/__init__.py:2:1: F401 '.views.ViewSet' imported but unused
./build/lib/laviewset/__init__.py:2:1: F401 '.views.HttpMethods' imported but unused
./laviewset/routes.py:138:1: W391 blank line at end of file
./laviewset/__init__.py:1:1: F401 '.routes.Route' imported but unused
./laviewset/__init__.py:2:1: F401 '.views.ViewSet' imported but unused
./laviewset/__init__.py:2:1: F401 '.views.HttpMethods' imported but unused
./tests/test_routes.py:3:1: F403 'from typing import *' used; unable to detect undefined names
./tests/test_routes.py:16:16: F405 'cast' may be undefined, or defined from star imports: typing
./tests/test_routes.py:73:36: F405 'cast' may be undefined, or defined from star imports: typing
./tests/test_routes.py:119:1: W391 blank line at end of file
./tests/test_views.py:307:1: W391 blank line at end of file

```

--- a/laviewset/__init__.py
+++ b/laviewset/__init__.py
@@ -1,2 +1,10 @@
+from typing import Tuple
+
 from .routes import Route
 from .views import ViewSet, HttpMethods
+
+__all__: Tuple[str] = (
+    'Route',
+    'ViewSet',
+    'HttpMethods'
+)

--- a/laviewset/routes.py
+++ b/laviewset/routes.py
@@ -134,5 +134,3 @@ def get_view_attrs(o: Callable[..., Any]) -> ViewAttrs:
             "i.e. if routes.is_view(callable) == True."
         )
     return getattr(o, _VIEW_ATTRS)
-
-

--- a/tests/test_routes.py
+++ b/tests/test_routes.py
@@ -1,6 +1,6 @@
 from __future__ import annotations
 
-from typing import *
+from typing import cast
 
 import pytest
 from aiohttp import web
@@ -112,8 +112,3 @@ def test_wrapping(base_router):
     assert view_attrs.path == test_route.path + ''
     assert view_attrs.method == _METHOD
     assert view_attrs.routedef_kwargs == _ROUTEDEF_KWARGS
-
-
-
-
-

--- a/tests/test_views.py
+++ b/tests/test_views.py
@@ -303,5 +303,3 @@ def test_registered_routedef(mock_webroute, base_route):
     assert call_args[0] == 'GET'
     assert call_args[1] == '/test'
     assert call_kwargs['z'] == 10
-
-

